_rules/auto-play-audio-does-not-exceed-3-seconds-aaa1bf.md

```diff
@@ -76,15 +76,15 @@ The `<video>` element audio output does not last longer than 3 seconds.
 
 #### Failed Example 1
 
-The `<audio>` auto-plays for more than 3 seconds.
+The `<audio>` `autoplay's` for more than 3 seconds.
```

Comment:
I think that "auto-play" was used as a verb, rather than as an attribute name. This is particularly visible on that case where the conjugation (final 's') make the whole syntax look weird.

I'd rather go for "automatically play" if you want to disambiguate "auto-play" from the `autoplay` attribute.

Reply:
In most places where just `auto-play` is used, it is rather confusing, hence the change. 

In the case of a plural, I am happy to change it to automatically play. I will await a few more people to have an opinion, before changing this.

Reply:
Yes, I agree that "auto-play" was a bit confusing. I think most places it was used as a verb and should be fixed as "automatically play" rather than "`autoplay`". Best w%ould be to have @Brynanders opinion on that since he wrote the rules…

Reply:
I am in favour of using "automatically play" or "play automatically" in stead of autoplay, to avoid confusion.

In this case, "autoplay's" is incorrect and should be "autoplays" without the apostrophe.

Refering to my previous comment, the backticks should be removed where autoplay is used as a verb and does not refer to the attribute.

Reply:
@jkodu @Jym77 @DagfinnRomen I agree with "automatically play(s)" or "play(s) automatically."